Tests/ImagineEngineTests/LabelTests.swift

```diff
@@ -122,4 +122,24 @@ final class LabelTests: XCTestCase {
         XCTAssertEqual(labelBClickCount, 1)
         XCTAssertEqual(clickedLabels, [labelB, labelA, labelA])
     }
+
+    func testScaling() {
+        let label = Label(text: "Hello world")
+
+        let upscaleFactor: Metric = 2.5
+        let downscaleFactor: Metric = 0.4
+
+        // Upscale the label
+        label.scale = upscaleFactor
+
```

Comment:
Very very nitpicky: But how about removing this newline since that's what you've done for the other statements? Also makes it nice and easy to see what asserts belong to what expressions 😉

Reply:
Fixed.